kokoro/linux-clang-release-bazel/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright (c) 2019 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Linux Build Script.
+
+# Fail on any error.
+set -e
+# Display commands being run.
+set -x
+
+CC=clang
+CXX=clang++
+SRC=$PWD/github/SPIRV-Tools
+
+cd $SRC
+./utils/git-sync-deps
```

Comment:
Our bots use the tip-of-tree for all of our dependencies.  We considered changing it (see https://github.com/KhronosGroup/SPIRV-Tools/pull/2345), but we need an autoroller first.  We need to be sure the deps file does not fall too far behind.

I know Dan asked for this, but I think we should do what the other bots do.

Reply:
Done